aries_cloudagent/config/argparse.py

```diff
@@ -559,6 +559,13 @@ def add_arguments(self, parser: ArgumentParser):
             This must be set if running in no-ledger mode.  Overrides any\
             specified ledger or genesis configurations.  Default: false.",
         )
+        parser.add_argument(
+            "--ledger-keepalive",
+            default=5,
+            type=ByteSize(min_size=5),
```

Comment:
This looks like a good thing to add, but I don't think the type is ByteSize (just int).

Reply:
I agree with you about the type int. I changed it. 
Pay attention, the ByteSize is used also for "max-outbound-retry" params, I suppose for the same reason I used it: the lower and upper bound limit embedded check. Could an utility class with the same embedded check for the int type could be helpful for the parser? Maybe IntegerValue could be added.